specification/maintenance/resource-manager/Microsoft.Maintenance/preview/2021-04-01-preview/Maintenance.json

```diff
@@ -0,0 +1,2373 @@
+        "operationId": "PublicMaintenanceConfigurations_List",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ListMaintenanceConfigurationsResult"
+            }
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/MaintenanceError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "PublicMaintenanceConfigurations_List": {
+            "$ref": "./examples/PublicMaintenanceConfigurations_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Maintenance/publicMaintenanceConfigurations/{resourceName}": {
+      "get": {
+        "tags": [
+          "PublicMaintenanceConfigurations"
+        ],
+        "summary": "Get Public Maintenance Configuration record",
+        "operationId": "PublicMaintenanceConfigurations_Get",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "description": "Resource Identifier",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MaintenanceConfiguration"
+            }
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/MaintenanceError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PublicMaintenanceConfigurations_GetForResource": {
+            "$ref": "./examples/PublicMaintenanceConfigurations_GetForResource.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{providerName}/{resourceParentType}/{resourceParentName}/{resourceType}/{resourceName}/providers/Microsoft.Maintenance/applyUpdates/{applyUpdateName}": {
+      "get": {
+        "tags": [
+          "ApplyUpdates"
+        ],
+        "summary": "Track Updates to resource with parent",
+        "description": "Track maintenance updates to resource with parent",
+        "operationId": "ApplyUpdates_GetParent",
```

Comment:
The name of this tag and operation group is weird. Typically should be noun to provide context. It will show up as class in SDKs. How does ResourceUpdate sound?

Reply:
Agreed. this can be breaking change ( we already have these operation and tag in older). we prefer to keep Operation/tag as it is.